app/server/appsmith-server/src/main/java/com/appsmith/server/dtos/CacheableApplicationJson.java

```diff
@@ -0,0 +1,13 @@
+package com.appsmith.server.dtos;
+
+import lombok.Data;
+
+import java.time.Instant;
+
+@Data
+public class CacheableApplicationJson {
+
+    ApplicationJson applicationJson;
+
+    Instant timeStamp;
```

Comment:
Nit: Minor suggestion to rename the variable to be explicit. Same with the other template POJO.

```suggestion
    Instant cacheExpiryTime;
```